source/wrapper/test/Wrapper.js

```diff
@@ -129,7 +129,12 @@ contract('Wrapper', async ([aliceAddress, bobAddress, carolAddress]) => {
       result = await getResult(swapContract, result.tx)
       emitted(result, 'Swap')
 
-      ok(await balances(wrapperAddress, [[tokenDAI, 0], [tokenWETH, 0]]))
+      ok(
```

Comment:
if we're pushing these yarn lint changes to, for example, `wrapper`. Does `@airswap/wrapper` need bumping?